manager/src/lib.rs

```diff
@@ -691,7 +690,7 @@ impl CronManager {
     /// If no offset, Returns current slot based on current block height
     /// If offset, Returns next slot based on current block height & integer offset
     /// rounded to nearest granularity (~every 1.6 block per sec)
-    fn get_slot_id(&self, offset: Option<u64>) -> u128 {
+    pub fn get_slot_id(&self, offset: Option<u64>) -> u128 {
```

Comment:
making this (and `get_slot_from_cadence`) public? Seems that we'd want to make sure we're returning `U128` instead of `u128`. When I did the sweep through I didn't change these since they were purely internal, private methods

Reply:
yeah, lets drop pub